api/v3/clairpb/clair.proto

```diff
@@ -44,17 +44,39 @@ message Vulnerability {
   repeated Feature affected_versions = 8;
 }
 
+message Detector {
```

Comment:
Please review: API is changed to have Detector attached to every single feature. Maybe this is not desirable and we should update the RPC endpoint to let the user to specify which detector result they want and whether to return the detectors associated with features.

Reply:
Keep the API design and don't add additional filtering.